### PR TITLE
Fix issue where AppTP can't be turned off

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/72649045549333/1201931173358115/f

### Description
But was introduced in #1763 where AppTP can't no longer be turned OFF

### Steps to test this PR

_Verify appTP can be turned off_
- [ ] install from this branch
- [ ] install another 3rd party VPN, eg. Netguard
- [ ] enable ApptP
- [ ] disable ApptP
- [ ] verify AppTP is disabled
- [ ] re-enable AppTP and verify it re-enables
- [ ] re-run the tests in #1763 to verify they also work (you'll need the 3rd party VPN for that)

### CODE REVIEW

**Comment by aitorvs** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #1797** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/1797" target="_blank">Graphite</a>  (this PR)

This comment was auto-generated by [Graphite](https://graphite.dev?utm_source=gh).

File: vpn/src/main/java/com/duckduckgo/mobile/android/vpn/network/RealVpnDetector.kt [hunk 1]
```diff
@@ -32,28 +28,12 @@ import javax.inject.Inject
...
-        val connectivityManager = context.applicationContext.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
-        val activeNetwork = connectivityManager.activeNetwork
-        return connectivityManager.getNetworkCapabilities(activeNetwork)?.hasTransport(NetworkCapabilities.TRANSPORT_VPN) ?: false
-    }
-
-    @TargetApi(Build.VERSION_CODES.LOLLIPOP_MR1)
-    private fun detectVpnLollipop(): Boolean {
         val connectivityManager = context.applicationContext.getSystemService(Context.CONNECTIVITY_SERVICE) as ConnectivityManager
-        return connectivityManager.activeNetworkInfo?.type == ConnectivityManager.TYPE_VPN
+        return connectivityManager.allNetworks
```
Comment (malmstein) [inline vpn/src/main/java/com/duckduckgo/mobile/android/vpn/network/RealVpnDetector.kt:35]: This is not the same as what we had before. This loops through all the networks tracked by the framework, as opposed to the active one, which is what we want.

Comment (malmstein) [inline vpn/src/main/java/com/duckduckgo/mobile/android/vpn/network/RealVpnDetector.kt:35]: val activeNetwork = connectivityManager.activeNetwork
        return connectivityManager.getNetworkCapabilities(activeNetwork)?.hasTransport(NetworkCapabilities.TRANSPORT_VPN) ?: false

**Review by malmstein — CHANGES_REQUESTED** [review]
we've changed what the detectVPN method does

**Review by malmstein — APPROVED** [review]
I believe you @aitorvs

**Pull request merged**